client-api/src/main/kotlin/org/ostelco/prime/client/api/store/SubscriberDAOImpl.kt

```diff
@@ -98,6 +98,20 @@ class SubscriberDAOImpl(private val storage: Storage, private val ocsSubscriberS
 
     }
 
+    override fun getMsisdn(subscriptionId: String): Either<ApiError, String> {
+        var msisdn: String? = null
+        try {
+            msisdn = storage.getSubscription(subscriptionId)
```

Comment:
That `storage.getSubscription()` should return the MSISDN is not that obvious. Maybe the "storage" class should be extended with a `getMsisdn()` method?

Reply:
currently subscription is` msisdn` ?

Reply:
Done